src/renderers/shaders/ShaderChunk/cube_uv_reflection_fragment.glsl

```diff
@@ -1,5 +1,9 @@
+#define DISABLE_CUBE_UV_MIPMAP_INTERPOLATION
```

Comment:
Maybe we can remove the commented code by now and restore from a previous state if needed. Just want to keep the code tidy.